raiden/tests/integration/test_channel_to_self.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from ethereum.tester import TransactionFailed
+
+#
+#  Tests if transaction attempting to create a new channel with both peers
+# set to a same address fails.
+#
+
+
+@pytest.mark.parametrize('number_of_nodes', [1])
+@pytest.mark.parametrize('channels_per_node', [0])
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+def test_channel_with_self(raiden_network, settle_timeout):
```

Comment:
Why make a new test file for this? Please add it to one of the other test files.